sensor_msgs/msg/AngularVelocity.msg

```diff
@@ -0,0 +1,4 @@
+std_msgs/Header header
+
+geometry_msgs/Vector3 angular_velocity
+float64[9] angular_velocity_covariance # Row major about x, y, z axes
```

Comment:
We should probably copy the comment from the `sensor_msgs/Imu` message here:

https://github.com/ros/common_msgs/blob/20a833b56f9d7fd39655b8491a2ec1226d2639b3/sensor_msgs/msg/Imu.msg#L5-L8